.github/workflows/build-and-publish.yml

```diff
@@ -0,0 +1,65 @@
+name: Build and publish
+on: [push]
+version: 2
+jobs:
+  build:
+    runs-on: [self-hosted, Linux, AWS]
+    timeout-minutes: 10
+    steps:
+      - name: Persist Workspace
+        uses: Brightspace/third-party-actions@actions/cache
+        with:
+          path: |
+            ./*
+          key: worksapce-v1-${{ github.sha }}
+
+      - name: Checkout
+        uses: Brightspace/third-party-actions@actions/checkout
+
+      - run:
+          name: Authenticate with Artifactory
+          command: curl -ks -u$DOCKER_USER:$DOCKER_PASS "https://d2lartifacts.jfrog.io/artifactory/api/npm/npm-local/auth/d2l" -o ~/project/.npmrc
+
+      - run:
+          name: Start docker containers
+          command: docker-compose -f ./dev/docker-compose.yml up -d dynamodb
+
+      - name: Setup Node
+        uses: Brightspace/third-party-actions@actions/setup-node
+        with:
+          node-version: 14
+          cache: npm
+
+      - run:
+          name: Install Dependencies
+          command: npm ci
+
+      - run:
+          name: Run tests
```

Comment:
we can skip running tests since it only does `npm ci`

Reply:
I meant to write, npm run test. Using npm ci was a typo